### DISCUSSION TITLE
Eraser on the Wacom pen to work just like in Wacom driver.

### DISCUSSION THREAD

**Discussion by KoleckOLP** [OpenTabletDriver/OpenTabletDriver#3162] [General]
### Description

What I mean is that by flipping the pen and howering with the eraser would select the eraser tool in drawing software and drawing with eraser would erase.

Flipping back to tip would go back to brush.

This is how it works in the Wacom driver.

I tried Windows Ink Eraser Toggle and Hold and neither do this function.

I was trying on the Wacom CTH-480 with the original LP-180E pen.

### Acknowledgements

- [X] I have searched the existing issues and this new issue is not a duplicate of any.
- [X] I have written a concise and meaningful title.
- [X] I am on the latest version of OpenTabletDriver.

**Comment by jamesbt365** [comment]
You may be able to get it to work if you set the pen tip binding on the eraser binding. The "eraser key" bindings are more for pens that do not have erasers but you still want to use an eraser on them.

On a side note, plugins already have access to eraser reports, so it should be entirely up to them for what they do with them.

**Reply by KoleckOLP** [reply]
Oh yeah that does work, thank you for telling me!
